nerdlets/pathpoint-nerdlet/services/DataManager.js

```diff
@@ -1062,24 +1061,19 @@ export default class DataManager {
 
   Getmeasures(touchpoints_by_country) {
     const tpc = []; // Count Touchpoints totals by Stage
-    this.stages.forEach(stage => {
+    while (tpc.length < this.stages.length) {
       const rec = {
         traffic_type: 'traffic',
         num_touchpoints: 0,
-        average: 0,
         total_count: 0,
         steps_indexes: [],
-        total_steps: 0,
-        num_steps_over_average: 0,
-        max_congestion: 0,
+        total_congestion: 0,
         steps_max_cong: [],
-        above_avg: stage.percentage_above_avg,
-        steps_over_percentage_indexes: [],
         capacity_status: 'NO-VALUE',
         capacity_link: ''
       };
       tpc.push(rec);
-    });
+    }
```

Comment:
This section below is where I would handle logic to set the max to some value based on the MIN if the user does not have a MAX configured.  We do not want to require the user to have a MAX configured because it's going to break older configs.

My suggestion of (1.5 * MIN) was just an idea.  Think it through.

Reply:
Ok, sounds good for me.